WcaOnRails/app/assets/stylesheets/users.scss

```diff
@@ -19,3 +19,16 @@ input[type="radio"].center-vertically {
   vertical-align: middle;
   height: 100%;
 }
+
+.input-group.date.datetimepicker {
+  width: 100%;
+}
+
+.form-group {
```

Comment:
But we moved away from the button... (why was that again?)
Em 08/02/2016 17:35, "Jeremy Fleischman" <notifications@github.com>
escreveu:

> In WcaOnRails/app/assets/stylesheets/users.scss
> <https://github.com/cubing/worldcubeassociation.org/pull/379#discussion_r52196648>
> :
>
> > @@ -19,3 +19,16 @@ input[type="radio"].center-vertically {
> >    vertical-align: middle;
> >    height: 100%;
> >  }
> > +
> > +.input-group.date.datetimepicker {
> > +  width: 100%;
> > +}
> > +
> > +.form-group {
>
> Also, what about adding the calendar button back and using one of the
> options I mentioned here: #379 (comment)
> <https://github.com/cubing/worldcubeassociation.org/pull/379#issuecomment-181185178>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cubing/worldcubeassociation.org/pull/379/files#r52196648>
> .
>